lib/ui/pages/coursetable/course_table_page.dart

```diff
@@ -656,13 +656,24 @@ class _CourseTablePageState extends State<CourseTablePage> {
                   icon: const Icon(Icons.add_outlined),
                   label: Text(R.current.details),
                 ),
+                TextButton.icon(
+                  onPressed: () => _showCameraPage(courseInfo),
+                  icon: const Icon(Icons.camera_alt_outlined),
+                  label: Text(R.current.details),
+                ),
               ]
             : [const SizedBox.shrink()],
       ),
       barrierDismissible: true,
     );
   }
 
+  Future<void> _showCameraPage(CourseInfoJson courseInfo) async {
```

Comment:
```suggestion
  void _showCameraPage(CourseInfoJson courseInfo) {
```

The function seems not only not return a `Future` but also not uses any `await` keyword inside, so suggest make it return `void` and remove `async`.